### DISCUSSION TITLE
how to add a mysql-database to aedes-broker

### DISCUSSION THREAD

**Discussion by Alanethm** [moscajs/aedes#673] [Q&A]
Hallo,
i need a help. i trying to build a broker base of aedes with connection to mysql-database.
i read readme but i was not sure if this is possible.

That why i am asking first if this is possible, when yes which dependency could i use for it.

thank for you help

**Comment by robertsLando** [comment]
If you want to create a persistence based on mysql database I suggest to check https://github.com/moscajs/aedes-persistence-mongodb as base project
